third_party/xla_client/pjrt_computation_client.h

```diff
@@ -140,6 +143,35 @@ class PjRtComputationClient : public ComputationClient {
     std::shared_ptr<PjRtBuffer> buffer;
   };
 
+  struct PjRtShardedData : public Data {
+    PjRtShardedData(std::string device, Shape shape)
+        : Data(std::move(device), std::move(shape)) {}
```

Comment:
what's this constructor for?

Reply:
It's a base constructor we should implement because we are implementing Data.

Reply:
It seems wrong that we construct a `PjRtShardedData` without shards

Reply:
Right, also we don't use it. Let me make a note of this, and see if we can avoid declaring it (e.g., delete) -- in the next PR.

Reply:
Ok, delete works.